src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setup() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size 메서드를 활용해 Set의 크기를 확인하는 테스트")
+    void sizeTest() {
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

다음과 같이 사용할 수도 있겠네요 😄